packages/features/ee/organizations/components/OrganizationAvatar.tsx

```diff
@@ -0,0 +1,39 @@
+import classNames from "@calcom/lib/classNames";
+import { getPlaceholderAvatar } from "@calcom/lib/defaultAvatarImage";
+import { Avatar } from "@calcom/ui";
+import type { AvatarProps } from "@calcom/ui";
+
+type OrganizationAvatarProps = Pick<AvatarProps, "size" | "imageSrc" | "alt" | "indicator"> & {
+  organizationName?: string | null;
+  organizationLogo?: string | null;
+};
+
+const OrganizationAvatar = ({
+  size,
+  imageSrc,
+  alt,
+  organizationName,
+  organizationLogo,
+}: OrganizationAvatarProps) => {
+  return (
+    <Avatar
+      size={size}
+      imageSrc={imageSrc}
+      alt={alt}
+      indicator={
+        (organizationLogo || organizationName) && (
+          <div
+            className={classNames("absolute bottom-0 right-0 z-10", size === "lg" ? "h-5 w-5" : "h-10 w-10")}>
```

Comment:
I think the sizes are wrong here  - Pushing a storybook file change so we can easily test this also 
<img width="1220" alt="CleanShot 2023-08-16 at 09 48 09@2x" src="https://github.com/calcom/cal.com/assets/55134778/2ac1b50d-5cde-4f34-b23e-54d34cf6d9c3">